tests/test_views.py

```diff
@@ -4,19 +4,54 @@
 from health_check.conf import HEALTH_CHECK
 from health_check.exceptions import ServiceWarning
 from health_check.plugins import plugin_dir
+from health_check.views import MediaType
 
 try:
     from django.urls import reverse
 except ImportError:
     from django.core.urlresolvers import reverse
 
 
+class TestMediaType:
+
+    def test_lt(self):
+        assert not MediaType('*/*') < MediaType('*/*')
+        assert not MediaType('*/*') < MediaType('*/*', 0.9)
+        assert MediaType('*/*', 0.9) < MediaType('*/*')
+
+    def test_str(self):
+        assert str(MediaType('*/*')) == "*/*;1"
+
+    def test_repr(self):
+        assert repr(MediaType('*/*')) == "MediaType: */*;1"
+
+    def test_eq(self):
+        assert MediaType('*/*') == MediaType('*/*')
+        assert MediaType('*/*', 0.9) != MediaType('*/*')
+
+    def test_from_string(self):
+        assert MediaType.from_string('*/*') == MediaType('*/*')
+        assert MediaType.from_string('*/*;0.9') == MediaType('*/*', 0.9)
+        assert MediaType.from_string('*/*;0.9') == MediaType('*/*', 0.9)
```

Comment:
It may also be worth having explicit tests for the various places you're allowed to have optional whitespace, starting with directly after the `;`, which I think is reasonably common.

Reply:
good lord you are right. This was wrong. I fixed it and added a bunch more tests. Please double check, before I release it.